concourse/pipelines/deploy.yml

```diff
@@ -267,12 +267,12 @@ resources:
     icon: docker
     source: &registry-source
       repository: frontend
-      aws_role_arn: arn:aws:iam::172025368201:role/pull_images_from_ecr_role
       aws_region: eu-west-1
-      aws_access_key_id: ((concourse-ecr-readonly-user_aws-access-key))
-      aws_secret_access_key: ((concourse-ecr-readonly-user_aws-secret-access-key))
-      aws_ecr_registry_id: "172025368201"
+      aws_ecr_registry_id: ((ecr_registry_id))
       variant: release
+      aws_ec2_credentials: true
+      aws_role_arns: ((concourse_deployer_role_arn)),((concourse_ecr_role_arn))
+      debug: true
```

Comment:
Presumably this is temporary - if so please add a TODO and consider linking to a card. Similarly above with things like the container registry name (which would presumably move to alphagov or something at some point).

Reply:
yes, I will remove this before I merge, I'm gathering opinions on whether the solution is the best way forward.

Reply:
LGTM